apps/sailfish/qml/components/ItemDetails.qml

```diff
@@ -189,6 +189,7 @@ Item {
             }
 
             ItemDetailRow {
+                visible: playcount != "-1"
```

Comment:
Should be just `-1`, playcount is an integer and thus shouldn't be an integer to string comparison (although JavaScript ignores the type, but I think Qt Creator even advices to use !== to include the type check).